torch_xla/csrc/ir_dump_util.cpp

```diff
@@ -266,7 +266,7 @@ std::string DumpUtil::ToHlo(c10::ArrayRef<torch::lazy::Value> values,
   auto is_sharded = ShardingUtil::SetHloSharding(&lowering_ctx);
   xla::XlaComputation computation = ConsumeValue(lowering_ctx.BuildXla());
 
-  static bool dump_post_optimizations =
+  bool dump_post_optimizations =
```

Comment:
I think this is intentional, all of the `XLA_xxxx` debug flag is static by design. What we usually do is control the env var in https://github.com/pytorch/xla/blob/master/test/run_tests.sh#L101

Reply:
Yea, but then I have to separate the test out. Otherwise, other tests will require it to be off.

Reply:
right.. I would recommend doing that. I am hesitating to make this env var different than every other debug flag which might cause confusion to user/developer.

Reply:
Sure.

Reply:
Done.